crates/solvers/openapi.yml

```diff
@@ -658,6 +659,12 @@ components:
             by the caller.
           allOf:
             - $ref: "#/components/schemas/DateTime"
+        surplusCapturingJitOrderOwners:
+          type: array
+          items:
+            $ref: "#/components/schemas/Address"
+          description: |
+            List of addresses on whose surplus will count towards the objective value of their solution (unlike other orders that were created by the solver).
```

Comment:
We need the same for the `gnosis/solvers` repo

Reply:
@squadgazzz good point! https://github.com/gnosis/solvers/pull/28